docs/fsharp/style-guide/formatting.md

```diff
@@ -556,6 +556,38 @@ type Volume =
 | ImperialPint of float
 ```
 
+If there is a single short union, you can omit the leading `|`.
+
+```fsharp
+type Address = Address of string
+```
+
+For a longer or multiline union, keep the `|`.
+
+```fsharp
+[<NoEquality; NoComparison>]
+type SynBinding =
+    SynBinding of
```

Comment:
```suggestion
    | SynBinding of
```

Reply:
Good spot! That was exactly the point of adding that sample 😅.